tests/test_classes.py

```diff
@@ -0,0 +1,119 @@
+"""
+This source file is part of an OSTIS project. For the latest info, see https://github.com/ostis-ai
+Distributed under the MIT License
+(See an accompanying file LICENSE or a copy at https://opensource.org/licenses/MIT)
+"""
+
+from sc_kpm import ScAddr, ScAgent, ScAgentClassic, ScEventType, ScModule, ScResult
+from sc_kpm.identifiers import CommonIdentifiers
+from sc_kpm.utils.action_utils import execute_agent, finish_action_with_status
+from tests.common_tests import BaseTestCase, server
+
+WAIT_TIME = 1
+
+
+class CommonTests(BaseTestCase):
+    def test_sc_agents(self):
+        class Agent(ScAgent):
+            ACTION_CLASS_NAME = "Stalin_3000"
+
+            def on_event(self, init_element: ScAddr, init_edge: ScAddr, action_element: ScAddr) -> ScResult:
+                finish_action_with_status(action_element, True)
+                return ScResult.OK
+
+        class AgentClassic(ScAgentClassic):
+            def on_event(self, init_element: ScAddr, init_edge: ScAddr, action_element: ScAddr) -> ScResult:
+                if not self._confirm_action_class(action_element):
+                    return ScResult.SKIP
+                finish_action_with_status(action_element, True)
+                return ScResult.OK
+
+        agent = Agent(event_class=Agent.ACTION_CLASS_NAME, event_type=ScEventType.ADD_OUTGOING_EDGE)
+        agent_classic = AgentClassic(classic_class_name := "classic")
```

Comment:
Why ScAgent can be created without passed identifier, but ScAgentClassic used with it? Can we create ScAgentClassic objects without passing arguments?

Reply:
Done